roles/netbootxyz/templates/menu/windows.ipxe.j2

```diff
@@ -19,7 +19,7 @@ choose win_version || goto windows_exit
 goto ${win_version}
 
 :arch_set
-iseq ${win_arch} x64 && set win_arch x86 || set win_arch x64
+iseq ${win_arch} x64 && set win_arch x86 && set win_arch arm64 || set win_arch x64
```

Comment:
This logic may not work well, as it's flipping between x86 and arm64, so x86 will probably never get set.

Reply:
Ah yeah good point, I'll add in automatic arch detection and a seperate menu as well